### PR TITLE
Publishes CE 3.18.3

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx [hunk 1]
```diff
@@ -184,4 +184,25 @@ Calico Enterprise 3.18.2 is now available. This release includes bug fixes and i
...
+
+### Improvements
+
+* one
+* two
+
+
+### Bug fixes
+
+* one
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: 1. Fixed an issue where the token login page is no longer working when an identity provider is configured.
2. Fix that workload egress traffic could be incorrectly handled as host-sourced traffic when DNS policy was used in conjuction with IPVS kube-proxy. 
3. Security updates.

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by radTuti** [comment]
@ctauchen this is ready to merge once @rene-dekker release notes comments are incorporated

File: calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx]: Added.

**Pull request merged**